src/preprocessor-plugin.js

```diff
@@ -61,37 +49,21 @@ module.exports = class TemplateImportPreprocessor {
       includeSourceMaps: true,
       includeTemplateTokens: true,
     };
-
-    this.templateLiteralConfig = {
-      getTemplateLocalsExportPath: '_GlimmerSyntax.getTemplateLocals',
-
-      importIdentifier: util.TEMPLATE_LITERAL_IDENTIFIER,
-      importPath: util.TEMPLATE_LITERAL_MODULE_SPECIFIER,
-
-      includeSourceMaps: true,
-      includeTemplateTokens: true,
-    };
   }
 
   toTree(tree) {
-    let compiled = stew.map(
-      tree,
-      `**/*.{js,gjs,ts,gts}`,
-      (string, relativePath) => {
-        let config = {
-          relativePath,
-          getTemplateLocalsRequirePath: this.getTemplateCompilerPath(),
-        };
-
-        if (relativePath.match(/\.(gjs|gts)$/)) {
-          Object.assign(config, this.templateTagConfig);
-        } else {
-          Object.assign(config, this.templateLiteralConfig);
-        }
+    let compiled = stew.map(tree, `**/*.{gjs,gts}`, (string, relativePath) => {
+      let config = {
+        relativePath,
+        getTemplateLocalsRequirePath: this.getTemplateCompilerPath(),
+      };
 
-        return preprocessEmbeddedTemplates(string, config).output;
+      if (relativePath.match(/\.(gjs|gts)$/)) {
```

Comment:
When will this condition ever not match?

Reply:
This conditional is directly tied to the line you changed above, turning `{js,gjs,ts,gts}` into just `{gjs,gts}`, so this isn't so much "refactoring" as "keeping the code consistent with the other changes you made". If you're planning on doing another pass in a separate PR, though, then I'm not going to lose sleep over it.